build.gradle

```diff
@@ -332,7 +342,7 @@ task source(dependsOn: src)
 task compile(dependsOn: src) 
 
 // Forcing srcBeautify on src step
-src.dependsOn srcBeautify
+//src.dependsOn srcBeautify
```

Comment:
Add this back in, for the latest version of master - there is already code in place (for this branch) to skip the step for windows.

Reply:
uncomment in https://github.com/picoded/JavaCommons-pdf/pull/1/commits/43f59fd2c2a51a86e482266c63cf264a1e2e7f0d